packages/proxmark3/build.sh

```diff
@@ -5,6 +5,14 @@ TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
 TERMUX_PKG_VERSION="1:4.13441"
 TERMUX_PKG_SRCURL=https://github.com/RfidResearchGroup/proxmark3/archive/v${TERMUX_PKG_VERSION:2}.tar.gz
 TERMUX_PKG_SHA256=49c4f1854b364aa7ea7083581351f867128e71ea783d0ecd71fc41bcf7f63584
+TERMUX_PKG_DEPENDS="libbz2, libc++, readline"
+TERMUX_PKG_BUILD_IN_SRC="true"
+TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
+
+termux_step_post_configure() {
+        LDLIBS="-L${TERMUX_PREFIX}/lib" INCLUDES_CLIENT="-I${TERMUX_PREFIX}/include" CFLAGS="-I${TERMUX_PREFIX}/include"
+        TERMUX_PKG_EXTRA_MAKE_ARGS="client -j $TERMUX_MAKE_PROCESSES CC=$CC CXX=$CXX LD=$CXX cpu_arch=generic SKIPREVENGTEST=1 SKIPQT=1 SKIPBT=1 SKIPPTHREAD=1 SKIPGPU=1 PLATFORM=PM3GENERIC"
+
```

Comment:
forgot an end bracket here...
also... why is this repeated again in termux_step_make ??